### DISCUSSION TITLE
[Help]: I am trying to use the Omada docker container on my Synology NAS, but appears to be 32-bit?

### DISCUSSION THREAD

**Discussion by AccelMotion** [mbentley/docker-omada-controller#459] [Help]
### Controller Version

v.5.7.4

### Describe Your Issue or Question

I am trying to run the Omada controller under Docker on my Synology DS412+. I followed all the directions but still get connection refused on 8088 (i put the container in host mode, verified the folders and the environment variables). 

Before I dump all my logs on here, I checked that my architecture was correct, but noticed that the CPU on this synology DS412+ is a Atom D2700... which the internets say is a 32-bit processor... does that mean that none of the images on here will actually work? There are no error messages from the Docker logs about the image provided, so I am not sure if it's able to work on this NAS. 
Can you verify the image architectures and whether it should run?
david

### Expected Behavior

I expect the auto-selected pull to load and run on my chosen NAS. 

### Steps to Reproduce

pull "latest"
have it run correctly on Atom D2700

### How You're Launching the Container

```plain
I used the Docker App under DSM 6.2.4 and associated instructions.

If the arch is right, then I'm happy to open another ticket or extend this with more data about how my running Docker/Omada does not respond on 8088.
```

### Container Logs

```plain
Can provide later
```

**Comment by mbentley** [comment]
It looks like the Intel Atom D2700 may support 32 and 64 bit but it is possible that the OS that the Synology DS412+ has is 32 bit which would restrict it from being able to use an amd64 based image.

The biggest blocker to running a 32 bit version of this image is that MongoDB no longer supports 32 bit operating systems so it's basically impossible to support.

If you could provide the output of a few commands, that would help validate what the current situation is:
`uname -a`
`docker info`
